services/src/pro/users/userdb.rs

```diff
@@ -121,6 +125,56 @@ pub trait UserDb: Send + Sync {
         quota_used_updates: I,
     ) -> Result<()>;
 
+    /// Log quota usage (computation units)
+    ///
+    /// # Errors
+    ///
+    /// This call fails if database cannot be accessed
+    ///
+    // TODO: move this method to some AdminDb?
```

Comment:
why?

Reply:
the `UserDb` was initially designed to provide users access to their own metadata, like name, session, quota, etc. Now we also use it for stuff like updating the quota from the QuotaTracking module which is something only the system/admin should do. So all the methods that don't really fit in the UserDb have this comment...